src/style.css

```diff
@@ -1 +1,18 @@
 /* styles go here */
+/* * {
```

Comment:
You should never keep redundant comments in your code - they may be confusing for those who read it